domains/bhushann.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "This subdomain is a redirection to my personal portfolio",
+  "repo": "https://github.com/bhushankhopkarr/bhushankhopkar.github.io",
+  "owner": {
+    "username": "BhushanKhopkar",
+    "email": "bhushankhopkar06@gmail.com",
+    "twitter": "bhushankhopkrr"
+  },
+  "record": {
+    "CNAME": "bhushankhopkar.github.io"
```

Comment:
404 GitHub page, please make sure the site actually shows content.

Reply:
i dont know what the actual issue is. it is getting redirected to bhushankhopkarr.is-a.dev

Reply:
Probably since you already tried connecting the domain. In your websites repo remove the `CNAME` file and it should resolve that issue.

Also after that, can you give a link/preview of your website?

Reply:
i have made the changes right now can we meet on discord if you dont mind? i need further assistance on this